README.md

```diff
@@ -54,8 +67,8 @@ Minimum: 2 GB RAM (3+ GB recommended)
 Minimum: 2 GB
  
 ### Optional Software
- * Matlab® (version 2013b or later - 32bit)
- * R® (versions 3.3 or later - 32bit) 
+ * Matlab® (version 2017b or later)
```

Comment:
Matlab >=2017b exists in 64 bit **only**, so no need to say that Matlab is required in 64 bit